Users/coafile.rst

```diff
@@ -39,6 +39,10 @@ wide coafile which has the lowest priority. The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.
 
+It can be used to define the type of files you usually don't want to lint
+like minified files and backup files (e.g. ``*.min.js``, ``*.orig``)
```

Comment:
you have disregarded an earlier review comment
https://github.com/coala/documentation/pull/481#discussion_r148798880

Reply:
ohh ya rebasing error :P sorry.

Reply:
Done